src/messagingSummaryItem/index.tsx

```diff
@@ -9,14 +9,20 @@ interface MessagingSummaryItemProps {
   readonly url: string
   readonly pictureUrl: string
   readonly label: string
-  readonly subLabel: string
+  readonly subLabel: string|JSX.Element
   readonly timeLabel: string
   readonly hasUnreadMessages: boolean
 }
 
 const UNREAD_COLOR = color.primaryText
 const READ_COLOR = color.secondaryText
 
+const generateSubLabel = (subLabel:string|JSX.Element) : JSX.Element => (
+    <span className={'kirk-messaging-summary-item-sub-label'}>
```

Comment:
convert to className value to string  `className="kirk-messaging-summary-item-sub-label"`